pkgs/applications/networking/cluster/nerdctl/default.nix

```diff
@@ -27,25 +22,39 @@ buildGoModule rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  buildFlagsArray = [
-    "-ldflags="
-    "-w"
-    "-s"
-    "-X github.com/AkihiroSuda/nerdctl/pkg/version.Version=v${version}"
-    "-X github.com/AkihiroSuda/nerdctl/pkg/version.Revision=<unknown>"
-  ];
+  buildFlagsArray =
+    let
+      t = "github.com/AkihiroSuda/nerdctl/pkg/version";
+    in
+    [
+      "-ldflags="
+      "-w"
+      "-s"
+      "-X ${t}.Version=v${version}"
+      "-X ${t}.Revision=<unknown>"
+    ];
```

Comment:
```suggestion
    [
      "-ldflags="
      "-w"
      "-s"
      "-X github.com/AkihiroSuda/nerdctl/pkg/version.Version=v${version}"
      "-X github.com/AkihiroSuda/nerdctl/pkg/version.Revision=<unknown>"
    ];
```

Reply:
this t = "gopath" shortening is pretty standard across nixpkgs. Either way reverted